hibernation-setup-tool.c

```diff
@@ -1630,11 +1637,6 @@ static int handle_post_systemd_suspend_notification(const char *action)
 
 static int handle_systemd_suspend_notification(const char *argv0, const char *when, const char *action)
 {
-    if (!getenv("SYSTEMD_SLEEP_ACTION")) {
-        log_fatal("These arguments can only be used when called from systemd");
-        return 1;
-    }
-
```

Comment:
Removed because this environment variable was not getting set during hibernate/resume, prohibiting hooks from running.